modules/tinymce/src/plugins/table/test/ts/browser/ui/TableClassListButtonsTest.ts

```diff
@@ -48,36 +52,35 @@ describe('browser.tinymce.plugins.table.ui.TableClassListButtonsTest', () => {
     assert.lengthOf(events, classes.length, 'Command executed successfully.');
   };
 
-  it('TINY-7476: Ensure that the checkmark appears for a single class', async () => {
+  const pPerformToggleClassSetup = async (classList: string[], trueMarked: number, falseMarked: number, toolbar: boolean) => {
     const editor = hook.editor();
     const sugarContainer = SugarBody.body();
 
     setEditorContentTableAndSelection(editor, 1, 1);
-    await pAssertNoCheckmarksInMenu(editor, 'Table styles', 2, sugarContainer);
-    toggleClasses(editor, [ 'a' ]);
+    await pAssertNoCheckmarksInMenu(editor, 'Table styles', 2, sugarContainer, toolbar);
+    toggleClasses(editor, classList);
 
     const expected = {
-      '.tox-menu': 1,
-      '.tox-collection__item[aria-checked="true"]': 1,
-      '.tox-collection__item[aria-checked="false"]': 1
+      '.tox-menu': toolbar ? 1 : 2,
+      '.tox-collection__item[aria-checked="true"]': trueMarked,
+      '.tox-collection__item[aria-checked="false"]': falseMarked
     };
-    await pAssertMenuPresence(editor, 'There should be a checkmark', 'Table styles', expected, sugarContainer);
-  });
+    await pAssertMenuPresence(editor, `There should be ${trueMarked} checkmark(s)`, 'Table styles', expected, sugarContainer, toolbar);
+  };
 
-  it('TINY-7476: Ensure that the checkmark appears for two classes', async () => {
-    const editor = hook.editor();
-    const sugarContainer = SugarBody.body();
+  it('TINY-7476: Ensure that the checkmark appears for a single class', async () => {
```

Comment:
```suggestion
  it('TINY-7476: Ensure that the checkmark appears for a single class in toolbar', async () => {
```
just to be super clear? I had to track back the boolean to figure this out, so